### PR TITLE
Remove CI runs on `beta` channels

### PR DESCRIPTION
**Pull request by grtlr**
## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Review by Alex6323 — DISMISSED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,8 +32,7 @@ jobs:
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
-        rust: [stable, beta]
```
Comment (thibault-martinez) [inline .github/workflows/ci.yml:35]: Does it make sense to remove beta but keep nightly ?

Comment (thibault-martinez) [reply .github/workflows/ci.yml:35]: Or maybe I'm misunderstanding what L40 does actually

Comment (grtlr) [reply .github/workflows/ci.yml:35]: Yeah, I removed it

**Review by thibault-martinez — COMMENTED** [review]
Maybe you can reduce `no_std` to only `stable`, I don't think we gain anything from beta/nightly

**Review by samuel-rufi — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**